common/tools/dev-tool/src/azure-sdk-mcp-server/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure SDK MCP Server
+
+A Model Context Protocol (MCP) server implementation for the Azure SDK repository. This server is designed to facilitate the development and testing of tools that communicate using the MCP.
```

Comment:
Does this really belong in dev-tool or along side that can be invoked via dev-tool externally so it can version separately if need be?

Reply:
It could definitely be separate! I was a little hesitant to add another package to rush.json (we will likely need to do that in order to ensure dependencies are installed I think?) so maybe putting it under dev-tool would help avoid making the pnpm migration more difficult. Then, we can take that out into a separate package afterwards.

Or do you think I should make this a utility package and add a rush.json entry?